x-pack/plugins/timelines/server/search_strategy/timeline/utils/format_timeline_data.ts

```diff
@@ -5,39 +5,14 @@
  * 2.0.
  */
 
```

Comment:
Just broke up this file into separate util files. Happy to rename the folder to helpers if we want to maintain that naming pattern?

Reply:
`helpers.ts` files are typically co-located in the same directory as the React components they support (e.g. `index.tsx`) to:
1) keep business logic out of the React component
2) encourage the creation of functions that are:
- small
- composable
- easily-unit-testable

"helper" and "utility" functions have similar properties, but the `helpers.ts` convention is intended to indicate the functions primarily exist to support use case `1`:

```
1) keep business logic out of the component
``` 

even though `helpers` have a lot in common with "utility" functions, as described by use case `2`:

```
2) encourage the creation of functions that are:
  - small
  - composable
  - easily-unit-testable
```

If the functions are unlikely to be useful outside of the component currently consuming them, perhaps organizing them under a `helpers` directory makes sense, as it would align with the convention. On the other hand, if these existing functions feel like they are generally applicable / re-usable beyond the component, `utils` makes sense as a directory name. 🙂

Reply:
Thanks, that distinction is clear. I've updated it to use the `helpers/` directory instead and moved the directory into the factory directory to show closer ownership